src/Hyperlink/index.tsx

```diff
@@ -1,29 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import isRequiredIf from 'react-proptype-conditional-require';
 import { Launch } from '../../icons';
 import Icon from '../Icon';
 
-import withDeprecatedProps, { DeprTypes } from '../withDeprecatedProps';
-
 export const HYPER_LINK_EXTERNAL_LINK_ALT_TEXT = 'in a new tab';
 export const HYPER_LINK_EXTERNAL_LINK_TITLE = 'Opens in a new tab';
 
-const Hyperlink = React.forwardRef((props, ref) => {
-  const {
-    className,
-    destination,
-    children,
-    target,
-    onClick,
-    externalLinkAlternativeText,
-    externalLinkTitle,
-    variant,
-    isInline,
-    showLaunchIcon,
-    ...attrs
-  } = props;
+interface Props extends Omit<React.ComponentPropsWithoutRef<'a'>, 'href' | 'target'> {
```

Comment:
[clarification] `Hyperlink` uses `forwardRef` below, yet the `Props` interface is defining it as `React.ComponentPropsWithoutRef`; should it be using `ComponentPropsWithRef`?

Reply:
Good question! In this case, the `forwardRef` wrapper is overriding the type and injecting the `ref` prop anyways, so it doesn't really matter whether we use `WithoutRef` or `WithRef` here - the type that will be seen elsewhere in the code is the same. But I think it makes sense to use `withRef` since we are dealing with refs, and it's perhaps less confusing that way. So I just pushed a small commit to change it.